issue.go

```diff
@@ -96,29 +96,58 @@ type Epic struct {
 	Done    bool   `json:"done" structs:"done"`
 }
 
+type Description struct {
+	Version int       `json:"version,omitempty"`
+	Type    string    `json:"type,omitempty"`
+	Content []Content `json:"content,omitempty"`
+}
+
+type Content struct {
+	Type    string    `json:"type,omitempty"`
+	Text    string    `json:"text,omitempty"`
+	Content []Content `json:"content,omitempty"`
+}
+
 // A description field may be a plain old string or a document structure. This
 // is extremely annoying as they have the same field name and cannot both unmarshal
 // to the same type without jumping through hoops.
-type Document string
+type Document struct {
+	Description Description `json:"description"`
+	Raw         []byte      `json:"raw"`
+}
+
+func (d *Document) String() string {
+	return string(d.Raw)
```

Comment:
The thinking for `String()` is that it would be just the text, with any formatting stripped. We'd use this for like truncated previews or things like that. If the content is just plain text, obviously we can just return that text, otherwise we'd have to accumulate the text from the document hierarchy.

Something along the following lines:

```go
type ContentSet []Content

func (c ContentSet) String() string {
  b := &strings.Builder{}
  for i, e := range c {
    if i > 0 {
      b.WriteString("  ")
    }
    b.WriteString(e.String())
  }
  return b.String()
}

func (c Content) String() string {
  return c.Text +" "+ c.Content.String()
}
```